#!/usr/bin/python3
def raise_exception_msg(message=""):
    """Raise the NameError exception with a message."""
    raise NameError(message)
